Demo/VaultDemo/VaultDemo/ViewController.swift

```diff
@@ -117,7 +117,7 @@ class ViewController: UIViewController {
   
     func performVaultTokenisation(card: CardInfo, cardRegistration: CardRegistration) {
         
-        MangopayVault.initialize(clientId: clientId, environment: .sandbox)
+        MangopayVault.initialize(clientId: clientId, environment: .sandbox, tenant: .eu)
```

Comment:
@elviva-mgp to avoid confusion here, no need to specify tenant in the example, `eu` is default, no?

Reply:
yeah